lib/src/storelike.rs

```diff
@@ -62,6 +62,10 @@ pub trait Storelike: Sized {
     /// If Include_external is false, this is filtered by selecting only resoureces that match the `self` URL of the store.
     /// WARNING: This could be very expensive!
     fn all_resources(&self, include_external: bool) -> ResourceCollection;
+    // get number of available jobs for number of CPUs
+    fn njobs(&self) -> usize{
```

Comment:
Ok. If we decided to re-use this pattern we can add it back. IMHO Rayon uses similar logic under the hood and it may be worth implementing a type suitable for par_iter.